examples/schema/getBinary/js.md

```diff
@@ -1,3 +1 @@
-```javascript
-
-```
+This is not fetchable from JavaScript, you can use the <a> tag with "href" property to download the binary in DOM.
```

Comment:
```suggestion
```
This is not fetchable from JavaScript, you can use the `<a>` tag with "href" property to download the binary in DOM.
```
```